### ISSUE TITLE
exclude borrowings in nexus export

### ISSUE THREAD

**Issue by LinguList** [digling/edictor#134]
This requires to allow the users to select one borrowing column before. The rule should be simply: if this column has any value, just exclude the cognate set (that is: treat it as a singleton).

The question is in principle, whether this is really needed, given that one could also simply have two COGID columns, one in which borrowings are included, one where they are given unique IDs.

**Comment by thiagochacon** [comment]
the COGID approach would be better, as borrowings must also be handled etymologically.

**Comment by LinguList** [comment]
Yes, and what you can already do when doing your nexus export is just choosing your preferred column of cogid (even partial cognates): you just open settings, type in the name of your column (it's autocomplete) in the Cognate IDs field, and refresh, and the cogids have changed to the other column. If you then export nexus, it will use the currently active ones (I think, if you have cross-semantic ones in one version, and concept-wise cognates in the other, it will even completely adhere to the user request, so you CAN in fact export cross-semantic cognates, but I'd need to check).
